.bandit.yml

```diff
@@ -0,0 +1,4 @@
+---
+skips: 
+  - "B101"
+  - "B601"
```

Comment:
I'm surprised that either of these needed to be disabled - at a glance, I don't see where we're using `assert` (B101) or `paramiko.exec_command` (B601).

Reply:
Good call, thanks. I had the `assert` function in the testing originally.